src/locales/af_ZA/metadata.ts

```diff
@@ -0,0 +1 @@
+export default { title: 'Afrikaans' };
```

Comment:
Should we use explicit type definitions?

```suggestion
export default <MetadataDefinitions> { title: 'Afrikaans' };
```

Reply:
Definitely do **not** use this syntax (`<type> value`), ever
It is TS 2.x historical and deprecated
Use `value as type` instead

Beside that: I would use what is used in the other locale definition files

Reply:
> It is TS 2.x historical and deprecated

I didn't get a deprecation warning though.

-------

The other files use something like this:

```suggestion
const metadata: MetadataDefinitions = {
  title: 'Afrikaans',
};
export default metadata;
```

Reply:
Alternatively we could rewrite all index.ts files to look like this:

```suggestion
export default { title: 'Afrikaans' } as MetadataDefinitions;
```

Which should it be?

Reply:
> The other files use something like this:
> 
> 
> 
> ```suggestion
> 
> const metadata: MetadataDefinitions = {
> 
>   title: 'Afrikaans',
> 
> };
> 
> export default metadata;
> 
> ```

This one

Reply:
Done